### ISSUE TITLE
about VideoLQ dataset

### ISSUE THREAD

**Issue by sunyclj** [ckkelvinchan/RealBasicVSR#83]
In VideoLQ datasets, there are only low-quality LQ images，none HR images. How can it be used as a test?

**Comment by electrodrago** [comment]
Yeah, there aren't any HR images because the aim is to test with NIQE metric (non-reference metric). The HRs only used for reference metrics like PSNR or SSIM, you can check that.
